server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1114,6 +1114,8 @@ private void closeConsistencyCheck() {
     }
   }
 
+  private boolean loggedErrorForTabletComparison = false;
```

Comment:
This does not need to be `volatile` because it's only read/set from a `synchronized` method, right?